test/testoutput/CMakeCache.txt

```diff
@@ -41,7 +41,7 @@ CMAKE_CXX_COMPILER_AR:FILEPATH=CMAKE_CXX_COMPILER_AR-NOTFOUND
 CMAKE_CXX_COMPILER_RANLIB:FILEPATH=CMAKE_CXX_COMPILER_RANLIB-NOTFOUND
 
 //Flags used by the CXX compiler during all build types.
-CMAKE_CXX_FLAGS:STRING=-isystem /opt/local/include:
+CMAKE_CXX_FLAGS:STRING=-isystem /opt/local/include
```

Comment:
should this file be committed to the repo at all?

Reply:
same